src/api/app/models/package.rb

```diff
@@ -1323,7 +1323,7 @@ def backend_build_command(command, params)
       # Note: This list needs to keep in sync with the backend code
       permitted_params = params.permit(:repository, :arch, :package, :code, :wipe)
 
-      Suse::Backend.post("/build/#{URI.escape(project.name)}?cmd=#{command}&#{permitted_params.to_h.to_query}", '')
+      Suse::Backend.post("/build/#{URI.escape(project.name)}?cmd=#{command}&#{permitted_params.to_h.to_query}")
```

Comment:
I think so, because this is assign to the backend request body: 
https://github.com/Ana06/open-build-service/blob/0a821509502fb794a5d125f4f8ec9e2af386f53a/src/api/lib/opensuse/backend.rb#L76

And the tests are not failing.